src/lemlib/movements/purepursuit.cpp

```diff
@@ -28,18 +28,19 @@ PurePursuit::PurePursuit(float trackWidth, const asset& path, float lookaheadDis
     // get the current competition state. If this changes, the movement will stop
     compState = pros::competition::get_status();
     // get the starting time
-    startTime = pros::millis();
+    startTime = pros::millis() * ms;
     // decode the asset
     // TODO: be able to pass in a buffer directly instead of covert to string
     std::string input(reinterpret_cast<char*>(path.buf), path.size);
     std::vector<std::string> lines = splitString(input, "\n");
     for (const std::string& line : lines) { // loop through all lines
         if (line == "endData" || line == "endData\r") break;
+        // todo make sure these units are correct
         std::vector<std::string> pointInput = splitString(line, ", "); // parse line
-        const float x = std::stof(pointInput.at(0)); // x position
-        const float y = std::stof(pointInput.at(1)); // y position
-        const float speed = std::stof(pointInput.at(2)); // speed
-        this->path.push_back({x, y, 0, speed}); // save data
+        const Length x = std::stof(pointInput.at(0)) * in; // x position
+        const Length y = std::stof(pointInput.at(1)) * in; // y position
+        const AngularVelocity speed = std::stof(pointInput.at(2)) * rpm; // speed
```

Comment:
```suggestion
        const LinearVelocity speed = std::stof(pointInput.at(2)) * inps; // speed
```
Change to correct unit